cypress/component/advanced/concatenated/spec.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { mount } from 'cypress-react-unit-test'
+
+describe('Concatenated 1', () => {
+  const Hello = () => <div>First</div>
+
+  it('renders', () => {
+    mount(<Hello />)
+    cy.contains('First').should('be.visible')
+  })
+})
```

Comment:
Fixed spec syntax. But what it is doing at all?

Reply:
ughh, this is a stray file I have committed accidentally, will remove